File: Java/MathematicalPrograms/OddOrEvenChecker.java
import java.util.Scanner;
public class OddOrEvenChecker {
	public static void main(String[] args) {
		 Scanner scan = new Scanner(System.in);	
		 System.out.println("Enter a number");
		 int numentered = scan.nextInt();
		 if (numentered > 0) {
			 System.out.println("Your number " + numentered + " is Positive");
		 }
		 else {
			 System.out.println("Your number " + numentered + " is Negative");
		 }
	}
}

File: C++/MathematicalPrograms/OddOrEvenChecker.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	int num;
	cout << "Enter a number";
	cin >> num;
	if (num%2 == 0) {
		cout << "Your number " << num << " is even";
	}
	else {
		cout << "Your number " << num << " is odd";
	}
}